### ISSUE TITLE
SNES mini problem saving games

### ISSUE THREAD

**Issue by Parche12** [ClusterM/hakchi2#959]
I have the snes mini classic, and in every rom that is not for the SNES (I mean, those for GBA, GB, SEGA Genesis) when I press reset, it comes with an error message and never I can save the state... 

How can I fix it? or is it a bug that has to be fixed?

Excuse me for my bad bad english

**Comment by Drakonas** [comment]
How much free space do you have on your SNES Mini Classic?

**Comment by Parche12** [comment]
Well, I have only 60 mb occupied... 

and that error only shows up with other than snes roms, if it would be free space problem, wouldnt I have it with snes roms too? 

EDIT: And another piece of info: I hacked my brothers NES mini classic the very same way, with the very same roms, and he can save data from other consoles roms, and I cant...

**Comment by Drakonas** [comment]
Have you tried holding down + select on the controller instead of pressing reset on the SNES?

**Comment by Parche12** [comment]
No I havent... but just did, and the same error shows up : C8 error... and pressing down + select made that the error screen stay put... I had to turn the SNES power button off, and then turn on again to the snes to work again.

**Comment by Drakonas** [comment]
Are you using ClusterM's cores or KMFDManic's cores. KMFDManic's cores have a lot of bugs... also did you install the extra_space mod?

**Comment by Parche12** [comment]
Im using clusters.... and not, I didnt install the extra_space mod.. should I ? Does it help saving games??

**Comment by ReyVGM** [comment]
You have too many games on the menu. Limit pages to 35 games to avoid this issue.

**Comment by Parche12** [comment]
Too many games where? Where can I choose that limitation option? 

EDIT: Maybe I found it? I choose to show 20 games per page (the minimum) Should it work now?

**Comment by Drakonas** [comment]
Why don't you try and see instead of asking us. We don't have your SNES.

**Comment by Parche12** [comment]
Please don't get angry, and sorry for the trouble, but this is my first time hacking anything... 
By the way, I did as ReyVGM suggested and it worked, now I can save on any game regardless of console. Thanks everybody

**Comment by TonyXL2** [comment]
There point is that this is not a message board. It is an issue tracker. So if you had a documented bug with screenshots, log msgs, etc, you would post that here. If you need help, troubleshooting, etc, there is a great community on Reddit and other message boards.

**Comment by Frank-5** [comment]
Isn't all this information in the FAQ or in the docs somewhere?  The very first thing I did based on what I read was to keep the games per page between 12 and around 30 to keep things running right and I've have had absolutely no problems. I added gb,gba, n64, SNES,  name, and RetroArch with a bunch of cores,  with all running great.  (Except no saves for the N64 Roms sadly).  
Anyway shouldn't this issue be closed now @Drakonas?
